base/gocb_dcp_feed.go

```diff
@@ -58,8 +59,9 @@ func StartGocbDCPFeed(collection *Collection, bucketName string, args sgbucket.F
 		return err
 	}
 	var collectionIDs []uint32
-	if collection.IsSupported(sgbucket.DataStoreFeatureCollections) {
-		collectionIDs = append(collectionIDs, collection.GetCollectionID())
+	if bucket.IsSupported(sgbucket.BucketStoreFeatureCollections) {
+		// FIXME (bbrks): why only 1 collection ID? How do we get the others in feed args?!
+		// collectionIDs = append(collectionIDs, collection.GetCollectionID())
```

Comment:
> **WARNING**
>FIXME @bbrks  

I'm not 100% sure what to do about this yet - but also not looked at it too much.

`FeedArguments` does not contain collection IDs, but instead a map of scope/collection names.